cql3/statements/update_statement.cc

```diff
@@ -254,6 +257,92 @@ void insert_prepared_json_statement::execute_operations_for_key(mutation& m, con
     }
 }
 
+
+static
+bytes get_key(const cql3::expr::expression& partition_key_restrictions) {
+    const auto* conjunction = cql3::expr::as_if<cql3::expr::conjunction>(&partition_key_restrictions);
+
+    if (!conjunction || conjunction->children.size() != 1) {
+        throw raft::broadcast_tables::unsupported_operation_error(fmt::format(
+            "partition key restriction: {}", partition_key_restrictions));
+    }
+
+    const auto* key_restriction = cql3::expr::as_if<cql3::expr::binary_operator>(&conjunction->children[0]);
+
+    if (!key_restriction) {
+        throw raft::broadcast_tables::unsupported_operation_error(fmt::format("partition key restriction: {}", *conjunction));
+    }
+
+    const auto* column = cql3::expr::as_if<cql3::expr::column_value>(&key_restriction->lhs);
+    const auto* value = cql3::expr::as_if<cql3::expr::constant>(&key_restriction->rhs);
+
+    if (!column || column->col->kind != column_kind::partition_key ||
+        !value || key_restriction->op != cql3::expr::oper_t::EQ) {
+        throw raft::broadcast_tables::unsupported_operation_error(fmt::format("key restriction: {}", *key_restriction));
+    }
+
+    return to_bytes(value->view());
+}
+
+static
+void prepare_new_value(raft::broadcast_tables::update_query& query, const std::vector<::shared_ptr<operation>>& operations) {
+    if (operations.size() != 1) {
+        throw raft::broadcast_tables::unsupported_operation_error(fmt::format("only operation \"value = X\" is supported"));
```

Comment:
if one tries to do:
`v1 = X, v2 = Y`
they will get an error
"only operation value = X is supported"
and get confused.

The clou here is that only a SINGLE operation (of any type) is supported right now. Please adjust the comment.

Reply:
Changed this to `"only one operation is allowed and must be of the form \"value = X\""`.